app/src/adapters/db/clients/postgres_client.py

```diff
@@ -123,6 +123,12 @@ def generate_iam_auth_token(aws_region: str, host: str, port: int, user: str) ->
     return token
 
 
+def get_database_url() -> str:
+    conf = get_db_config()
+    conn = get_connection_parameters(conf)
```

Comment:
Ah right! I forgot having to do this for pg_dump: https://github.com/navapbc/labs-decision-support-tool/blob/83318787328c4529225468adcda514e47d09cbfe/app/src/db/pg_dump_util.py#L148